src/YMenuButton.cc

```diff
@@ -216,6 +216,50 @@ YMenuButton::resolveShortcutConflicts()
 }
 
 
+YMenuItem *
+YMenuButton::findItem( std::vector<std::string> & path ) const
+{
+    return findItem( path.begin(), path.end(),
+                     itemsBegin(), itemsEnd() );
+}
+
+
+YMenuItem *
+YMenuButton::findItem( std::vector<std::string>::iterator path_begin,
+                       std::vector<std::string>::iterator path_end,
+                       YItemConstIterator begin,
+                       YItemConstIterator end ) const
+{
+    for ( YItemConstIterator it = begin; it != end; ++it )
+    {
+        YMenuItem * item = dynamic_cast<YMenuItem *>(*it);
+        // Test that dynamic_cast didn't fail
+        if ( !item )
+            return nullptr;
+
+        if ( item->label() == *path_begin )
+        {
+            if ( std::next(path_begin) == path_end )
+            {
+                // Only return items which can trigger an action.
+                // Intermediate items only open a submenu, so continue looking.
+                
+                if( item->hasChildren() )
+                    continue;
+
+                return item;
+            }
+            
+            // Look in child nodes and return if found one
```

Comment:
one was found?

Reply:
This was an intermediate state of that file that is now outdated. Please check the latest version.